File: backend/ConAlambreApi/Models/Usuario.cs
using System.ComponentModel.DataAnnotations;

namespace ConAlambreApi.Models
{
    public class Usuario{

      [Key]
      public int Id {get; set;}

      public string Nombre {get; set;}

      public string Apellido {get; set;}

      public string Email {get; set;}

      public string Telefono {get; set;}

      public string Avatar {get; set;}
    }
}

File: frontend/src/types/usuario.ts
export interface UsuarioResponse {
  id: number;
  nombre: string;
  apellido: string;
  email: string;
  telefono: string;
  rol: string;
  avatar: string;
}

export interface RegisterRequest {
  nombre: string;
  apellido: string;
  email: string;
  contrasena: string;
  telefono: string;
  rol?: string;
  avatar?: string;
}

export interface UpdateUsuarioRequest {
  nombre?: string;
  apellido?: string;
  telefono?: string;
  avatar?: string;
  rol?: string;
}

